ingress/controllers/nginx-third-party/utils.go

```diff
@@ -206,7 +201,18 @@ func getTcpServices(kubeClient *unversioned.Client, tcpServices string) []nginx.
 
 		namePort := strings.Split(svc, ":")
 		if len(namePort) == 2 {
-			tcpSvc := getService(kubeClient, namePort[0])
+			tcpSvc, err := getService(kubeClient, namePort[0])
+			if err != nil {
+				glog.Errorf("%s", err)
+				continue
+			}
+
+			// the exposed TCP service cannot use 80 or 443 as ports
+			if namePort[1] == "80" || namePort[1] == "443" {
```

Comment:
nit, suggest httpPort httpsPort const and using them here and in the error message